star-ui/src/index.css

```
@@ -149,475 +47,2402 @@
+.panel-note,
+.metric-label,
+.metric-value,
+.metric-detail,
+.health-row__value,
+.list-card__caption,
+.list-card__footer,
+.trace-card__header p,
+.trace-card__footer,
+.empty-state {
+  overflow-wrap: anywhere;
+  word-break: break-word;
```

Stylelint (17.7.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)